Python/compile.c

```diff
@@ -2155,6 +2156,41 @@ compiler_class(struct compiler *c, stmt_ty s)
     return 1;
 }
 
+static int
+check_is_arg(expr_ty e)
+{
+    if (e->kind != Constant_kind) {
+        return 1;
+    }
+    PyObject *value = e->v.Constant.value;
+    return (value == Py_None
+         || value == Py_False
```

Comment:
I wouldn't special case True and False, those are not normally identity checked.  I don't doubt that _someone_ may have create APIs returning multiple possible types that requires this to differentiate between "real" values and bools... but yuck.